src/lib/mongoBackend/mongoQueryContext.cpp

```diff
@@ -310,12 +310,12 @@ HttpStatusCode mongoQueryContext
                        limit,
                        &limitReached,
                        countP);
-    reqSemGive(__FUNCTION__, "ngsi10 query request", reqSemTaken);
```

Comment:
I didn't remember this part... we move reqSemGive in mongoQueryContext to cover a wide scope (in fact, the scope that actually correspond to the operation, not sure why was so "reduced" before this change). Nice.

NTC